settings.gradle

```diff
@@ -1,12 +1,31 @@
+buildscript {
```

Comment:
Shouldn't we use plugins DSL instead of buildscript?

Reply:
Actually, I think this should be in root build.gradle instead. From what I can see online, settings.gradle generally has just pluginManagement & dependencyResolutionManagement blocks. And under pluginManagement, seems like folks are generally just defining the repositories for plugins there.